### PR TITLE
correct spelling mistake

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

File: lib/inflecto.rb [hunk 1]
```diff
@@ -272,7 +272,7 @@ def self.tableize(input)
   #   Inflecto.classify("posts")        # => "Post"
   #
   #   # Singular names are not handled correctly:
-  #   Inflecto.classify("business")     # => "Busines"
+  #   Inflecto.classify("business")     # => "Business"
```
Comment (mbj) [inline lib/inflecto.rb]: does the code actually do this?

Comment (mbj) [reply lib/inflecto.rb]: I've seen you opened other PRs to other repos, so your activity could be based by spell check bot. This instance AFAIK actually documents a misspelling generated by the code.

Comment (EdwardBetts) [reply lib/inflecto.rb]: Sorry about that.

Comment (mbj) [reply lib/inflecto.rb]: np, had a positive intend I'm sure.

**Pull request closed without merging**